students/mathewcm/break_me.py

```diff
@@ -0,0 +1,16 @@
+# def test1():
+#    x = y
+
+# def test2():
+#    'now' + 5
+
+# def test3():
+#    if a > b
+
+def test4():
+   raise AttributeError
```

Comment:
Thanks @HarryMaher for your comment on the main thread. I think @HarryMaher meant this trick. Yes, we definitely wanted to see an example like `"blah".fake_attribute"`.

@HarryMaher , I'll not merge PRs without going through them next time, and I really appreciate your going over PRs with comments!